content/departments/engineering/managed-services/cloud-ops.md

```diff
@@ -0,0 +1,55 @@
+# Cloud Ops Dashboard infrastructure operations
+
+<!--
+Generated documentation; DO NOT EDIT. Regenerate using this command: 'sg msp operations generate-handbook-pages'
+-->
+
+This document describes operational guidance for Cloud Ops Dashboard infrastructure.
+This service is operated on the [Managed Services Platform (MSP)](../teams/core-services/managed-services/platform.md).
+
+If you need assistance with MSP infrastructure, reach out to the [Core Services](../teams/core-services/index.md) team in #discuss-core-services.
+
+## Service overview
+
+| PROPERTY     | DETAILS                                                                                                                    |
+| ------------ | -------------------------------------------------------------------------------------------------------------------------- |
+| Service ID   | [`cloud-ops`](https://github.com/sourcegraph/managed-services/blob/main/services/cloud-ops/service.yaml)                   |
+| Owners       | **cloud**                                                                                                                  |
+| Service kind | Cloud Run service                                                                                                          |
+| Environments | [prod environment](#prod-environment)                                                                                      |
+| Docker image | `us-central1-docker.pkg.dev/control-plane-5e9ee072/docker/apiserver`                                                       |
+| Source code  | [`github.com/sourcegraph/controller` - `cmd/apiserver`](https://github.com/sourcegraph/controller/tree/HEAD/cmd/apiserver) |
+
+## Environments
+
+### prod environment
```

Comment:
nit:

```suggestion
### `prod`: environment
```

Reply:
I want to avoid backticks here - it seems the heading anchoring behaviour varies a lot

Reply:
How about just `prod` for the top-level environment header? 😛 and leave everything else as-is? The `-` anchor didn't work for me in VS code.

It doesn't look pretty, but I'm leaning towards keeping the link-ability 😆

Reply:
sgtm